app/public/templates/tutorial.html

```diff
@@ -103,19 +103,31 @@
             id="blackboard"
             visible="true"
             geometry="primitive: plane; width: 18 ; height: 15"
-            material="color: #333333; shader: flat; transparent: false"
+            material="color: #1A1F26; shader: flat; transparent: false"
           >
           </a-entity>
           <a-entity
             id="returnButton"
-            close-popup='jsonData: {"width": "1", "height": "1", "depth": "0.01", "color": "red", "x": "-8.25", "y": "6.75", "z": "0.005"};'
+            close-popup='jsonData: {"width": "1", "height": "1", "depth": "0.01", "color": "red", "opacity": "0", "x": "-8.25", "y": "6.75", "z": "0.005"};'
             class="link"
-          ></a-entity>
+          >
+            <a-entity
+              id="xClose"
+              text="align: center; width: 1; wrapCount: 1; value: X"
+              position="0 0 0"
+            ></a-entity>
```

Comment:
Correct me if I'm wrong, but I think this is the logic where the "X" button gets rendered in the top right Dhruvin? So in regards to Aaron's comment, I don't know how you can really scale it down. Two ideas (but not sure if they'll work):
- change the X to x (ie. lower case). If they preserve that on render, it'll be smaller by nature 
- change the width attribute (and maybe add a height attribute if it exists) to make it smaller by constraint

Let me know if that works or not!

Reply:
Tried a few things. I think keeping it big X and just decreasing the width works the best here since it also keeps it centered within its parent element